File: tugas 1.java
public class Main { //1.main program
    public static void main(String[] args) {
        Integer angka;//3. tipe data
        int x = 20;
        int y = 18;
        String[] cars = {"Volvo", "BMW", "Ford", "Mazda"};;
        System.out.println("Hello world!");
        String name = "John";// tipe data
        System.out.println(name);
        angka = 5;//2.variabel
        //perulangan for dan array statis satu dimensi
        System.out.println(angka);
        for (int i=0; i<=3;i++){//4. for
            System.out.println(cars[i]);//8. array satu dimensi
        }
        if (x > y) {// if
            System.out.println("x is greater than y");
        }
        int b = 1;
        while (b <= 3){//while
            System.out.println("Hello World");
            b++;
        }
        int a=1;
        do{// do while
            System.out.println(a);
            a++;
        }while(a<=4);
        // array multi dimensi
        int[][] myNumbers = { {1, 2, 3, 4}, {5, 6, 7} };
        int z = myNumbers[1][2];
        System.out.println(z);

    }


}

File: tugas1.cpp
#include<iostream>
using namespace std;

int main(){
	int angka;
	string nama[3]={"nanda", "asep", "udin"};
	int myNumbers[3][4]= { {1, 2, 3, 4}, {5, 6, 7} };
	angka=5;
	for(int i=0;i<=2; i++){
		cout<<"nama: "<<nama[i]<<endl;
	}
	if(angka>4){
		cout<<"5 lebih dari 4"<<endl;
	}
	int b = 1;
        while (b <= 3){//while
            cout<<"Hello World"<<endl;
            b++;
        }
    int a=1;
        do{// do while
            cout<<a<<endl;
            a++;
        }while(a<=4);
        // array multi dimensi
        int z = myNumbers[1][2];
        cout<<"array yang dipanggil: "<<z;
	
}
